sparse_strips/vello_bench/src/data.rs

```diff
@@ -118,7 +119,9 @@ impl DataItem {
                 width: path.stroke_width as f64,
                 ..Default::default()
             };
-            paths.push(flatten::expand_stroke(path.path.iter(), &stroke, 0.25));
+            let mut stroke_ctx = StrokeCtx::default();
```

Comment:
Do we not want to reuse `StrokeCtx` for each `path` by moving it outside the loop?